v1.0/monitor-cockroachdb-with-prometheus.md

```diff
@@ -156,7 +156,7 @@ If you like, you can connect [Alertmanager](https://prometheus.io/docs/alerting/
 
     ~~~ shell
     $ prometheus --config.file=prometheus.yml \
-    --alertmanager.url=<hostname of machine running alertmanager>:9093
+    --alertmanager.url=<address of machine running alertmanager>:9093
```

Comment:
come to think of it, we may want to be consistent with `8.` below which mentions: `http://<hostname of machine running alertmanager>:9093`.